app/models/access_token.rb

```diff
@@ -29,4 +29,8 @@ def revoked?
   def revoke
     update revoked_at: Time.current if revoked_at.nil?
   end
+
+  def authorize?(required_scope)
```

Comment:
authorize は動詞なので、状態を確認するなら authorize *d* ? のほうがよさそう。
```suggestion
  def authorized?(required_scope)
```

Reply:
それもそうですね、修正します。